ivi-layermanagement-examples/EGLWLInputEventExample/src/main.cpp

```diff
@@ -89,7 +89,7 @@ int main(int argc, char **argv)
     // initialize eyes renderer
     if (!InitRenderer()){
         fprintf(stderr, "Failed to init renderer\n");
-        return -1;
+        goto Error;
```

Comment:
Good use of goto.  I approve... :-)
But the returned value changed to zero. This is just a demo program, so maybe you intended to ignore this, but it would be more correct to exit with non-zero value if an error was noticed.

Reply:
Thank you for review. Yes, I ignored it. It would be better if I return non-zero value